lib/ansible/modules/system/mount.py

```diff
@@ -74,9 +74,11 @@
       - C(absent) specifies that the device mount's entry will be removed from
         I(fstab) and will also unmount the device and remove the mount
         point.
+      - C(remounted) specifies that the device will be remounted for when you 
+        want to force a refresh on the mount itself (added in 2.9).
```

Comment:
'this will always returned changed=true' <= or similar should be made clear to user

Reply:
added